packages/fluentui/docs/src/examples/components/Grid/Variations/GridExampleRows.tsx

```diff
@@ -30,10 +30,21 @@ const images = [
 const GridExample = () => (
   <div>
     Grid with specified number or rows:
-    <Grid rows="2">{images}</Grid>
+    <Grid rows={2}>{images}</Grid>
     <br />
     Grid with explicitly specified rows:
-    <Grid rows="2fr repeat(2, 1fr)">{images}</Grid>
+    <Grid
+      styles={{
+        gridAutoFlow: 'column',
+        gridTemplateRows: '2fr repeat(2, 1fr)',
+        msGridRows: '2fr (1fr)[2]',
+      }}
+      variables={{
+        defaultColumnCount: undefined,
```

Comment:
What it does? Cruft?

Reply:
In the grid variables files we have `defaultColumnCount` set for 5 and because in the example we don't have `rows` props set it will get the value 5 since:

```typescript
const { rows, columns = !props.rows && defaultColumnCount } = props;
```

So it would not generate the expected result